README.md

```diff
@@ -46,6 +46,8 @@ To set up CoreNLP version 4.0.0, do `./setup_corenlp.sh` after changing `corenlp
 
 After installation, or if you already have CoreNLP installed, let BlaBla know where to find it using `export CORENLP_HOME=/path/to/corenlp`.
 
+CoreNLP also requires the [Java Developer Kit](https://www.oracle.com/java/technologies/javase-downloads.html) to be installed.
```

Comment:
Can we add a statement saying the user can check if JDK is installed or not by running the `javac -version` command?

Reply:
Definitely!